fontforge/python.c

```diff
@@ -5832,6 +5833,7 @@ static int PyFF_Glyph_CSetLayer(PyFF_Glyph *self, PyObject *value, int layeri, i
     }
 
     if ( PyErr_Occurred() != NULL ) {
+        free(ss);
```

Comment:
I'm not happy about this one. `ss` is only set via a call to `_SSFromLayer` or `_SSFromContour`. Those should either set an error and return NULL or return a proper `SplineSet`. If this is from a Coverity report rather than by inspection, we should track down the path that returns a non-NULL `SplineSet` pointer while setting an error. 

And anyway, SplineSets have sub-units and are freed with `SplinePointListsFree()` rather than just free.

Reply:
I removed the free instruction since it was only based on Coverity reports